.ci/jobs/defaults.yml

```diff
@@ -3,7 +3,7 @@
 ##### GLOBAL METADATA
```

Comment:
Review the line https://github.com/elastic/e2e-testing/pull/156/files#diff-c35ec66bcd810edcb08ef01abec77275R18 to avoid the indexing